netty/src/test/java/io/grpc/netty/NettyServerTest.java

```diff
@@ -91,4 +102,65 @@ public void getPort_notStarted() throws Exception {
 
     assertThat(ns.getPort()).isEqualTo(-1);
   }
+
+  @Test(timeout = 60000)
```

Comment:
We should not use timeout here, and instead use a JUnit4 timeout rule:

`@Rule public final Timeout globalTimeout = Timeout.seconds(30);`


The reason is that the timeout rule works correctly when debugging / stepping through code.